site/src/pages/WorkspacePage/WorkspacePage.test.tsx

```diff
@@ -92,16 +92,6 @@ afterAll(() => {
 })
 
 describe("WorkspacePage", () => {
-  it("shows a workspace", async () => {
```

Comment:
Removed this test because all the other tests are confirming a workspace is displayed.